src/components/NoteList.js

```diff
@@ -17,16 +19,29 @@ NoteItem.defaultProps = {
   title: PropTypes.string,
 };
 
-const Notes = props => (
-  <ul>{props.notes.map(note => <NoteItem key={note.id} {...note} />)}</ul>
-);
+class NoteList extends Component {
+  componentDidMount() {
+    this.props.fetchNotes();
+  }
+  render() {
+    return (
+      <ul>
+        {this.props.notes.map(note => <NoteItem key={note.id} {...note} />)}
+      </ul>
+    );
+  }
+}
 
-Notes.propTypes = {
+NoteList.propTypes = {
   notes: PropTypes.arrayOf(PropTypes.object),
+  fetchNotes: PropTypes.func,
 };
 
-Notes.defaultProps = {
+NoteList.defaultProps = {
   notes: PropTypes.arrayOf(PropTypes.object),
+  fetchNotes: PropTypes.func,
 };
 
-export default Notes;
+export default connect(state => ({ notes: state.notes }), { fetchNotes })(
```

Comment:
@laurelherting this is the redux function to hook the state up with the fetchNotes reducer Action. The NoteList object is passed in to the connection function. Thats what the connect()(NoteList) does.